ci-operator/step-registry/cucushift/upgrade/eus/pause-worker-mcp/cucushift-upgrade-eus-pause-worker-mcp-commands.sh

```diff
@@ -21,10 +21,30 @@ function debug() {
 
 function pause() {
     echo "Pausing worker pool..."
-    oc patch --type=merge --patch='{"spec":{"paused":true}}' machineconfigpool/worker
-    ret=$(oc get machineconfigpools worker -ojson | jq -r '.spec.paused')
+    oc patch --type=merge --patch='{"spec":{"paused":true}}' mcp/${PAUSED_MCP_LABEL}
+    if [ $? -ne 0 ]; then
+        echo "Failed to pause mcp"
+        return 1
+    fi
+
+    # set maxUnavailable for worker mcp
+    local running_worker running_worker_list running_worker_size
+    running_worker=$(oc get -l 'node-role.kubernetes.io/'${PAUSED_MCP_LABEL}'!=, node-role.kubernetes.io/master!=' nodes)
+    echo "Normal worker nodes:"
+    echo ${running_worker}
+    mapfile -t running_worker_list <<< $running_worker
+    running_worker_size=${#running_worker_list[@]}
+
+    echo "Set maxUnavailable ${running_worker_size} for worker mcp"
```

Comment:
I think `maxUnavailable` is not a MUST for the pause step, if we do not set it, the default value is `1`, that means, only 1 node in the mcp is being updating at the moment.

Reply:
I am ok to not set it